frontend/login_menu.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+// ======================================== Logowanie do strony ======================================== //
+
+let login_button = document.getElementById("submit_button");
+let user_login2 = document.getElementById("user_login");
+let user_password2 = document.getElementById("user_password");
+let login_status = document.getElementById("login_info");
+
+login_button.addEventListener("click", () => login_user(user_login,user_password));
+
+async function login_user(login, password){
+    let response = await fetch(`users.json`);
+    let obiekt = await response.json();
+        if (login.value === "admin" && password.value === "admin"){
+            login_status.value = "Chłop co wpisał admin admin xdddd"
+            return;
+        }
+        if (login.value === "" || password.value === ""){
+            login_status.value = "Uzupełnij pola logowania"
+            return;
+        }
+        for (let check of obiekt.account){
+        
+            if (check.login === login.value){
+                if(check.password === password.value){
+                    login_status.value = "Logowanie..."
```

Comment:
There is a lot of hard-coded values used for `login_status.value`. What do you think about creating an enum for them?
There are no native enums in JavaScript, but you could do something like that:
```
const LoginStatuses = {
  IN_PROGRESS: 'Logowanie',
  MISSING_INPUT: 'Uzupełnij pola logowania',
  INVALID_PASSWORD: 'Błędne hasło',
  INVALID_USER: 'Nie ma takiego użytkownika',
  EASTER_EGG:'Chłop co wpisał admin admin xdddd'
}
```

Then you can assign the value as follows: `login_status.value = LoginStatuses.IN_PROGRESS` etc.

Reply:
that was a good idea